spec/factories/events.rb

```diff
@@ -123,6 +123,12 @@
       end
     end
 
+    trait :with_enough_free_places do
+      after(:build) do |event|
+        event.max_participants = event.application_letters.size
```

Comment:
Besides the missing +1 at the end breaks the tests atm, I'm not really sure if its a good idea to have a trait like this at all. From my understanding, this works only if all applications were added before, e.g. in another trait. So if you put this trait before :with_diverse_open_applications or similar, it will not work as expected. Imo that could bring more problems than benefits for testing